source/index.html

```diff
@@ -1,12 +1,212 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
   </head>
-  <body>
+  <body class="page__body">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <header class="page-header">
+      <a class="page-header__logo" href="#" >
+        <img class="page-header__logo-image" src="../source/img/header_logo_pink.svg" alt="Логотип" width="74" height="23">
```

Comment:
```suggestion
        <img class="page-header__logo-image" src="img/header_logo_pink.svg" alt="Логотип" width="74" height="23">
```
1. Символы `../` в относительном пути к файлу указывают на то, что нужно перейти на один уровень вверх. Файлы html лежат в корне проекта, а изображения в папке `img`, на один уровень ниже относительно файлов html, следовательно символы `../` подставлять не нужно.
2. Папку `source` указывать не нужно, так как вся работа ведётся относительно неё и она является корнем проекта.